doc/classes/String.xml

```diff
@@ -392,6 +392,10 @@
 			<return type="String">
 			</return>
 			<description>
+				Escapes (encodes) a string to URL friendly format. Also referred to as 'URL encode'.
+				[codeblock]
+				print( "https://example.org/?escaped="+"Godot Engine:'docs'".http_escape() )
```

Comment:
There shouldn't be spaces after `(` and before `)` as per the GDScript style guide. And there should be spaces around operators like `+`.

I'm not sure the concatenation adds much to the example though, so I'd simplify it by using `https://example.org/?escaped=Godot Engine:'docs'"` directly.

Reply:
The concatenation is so it's not the complete URL that's encoded - only the part you'd encode in a real URL.
I'll fix the changes and try and see if I can get the squash to work this time

Reply:
Ah right, good point.